### PR TITLE
Loading page

### PR DESCRIPTION
**Pull request by avoesen**

### CODE REVIEW

File: components/Loading.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import PlateWheel from "/public/icons/svg/plate-wheel.svg";
+
+export default function LoadingComponent() {
+  return <RotatingSVG />;
+}
+const RotatingSVG = styled(PlateWheel)`
+  margin: 4rem auto;
+  /* margin-top: 2rem;
+  margin-bottom: 2rem; */
```
Comment (MerleMak) [inline components/Loading.js]:
```suggestion
```

File: components/Loading.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import PlateWheel from "/public/icons/svg/plate-wheel.svg";
+
+export default function LoadingComponent() {
+  return <RotatingSVG />;
+}
+const RotatingSVG = styled(PlateWheel)`
```
Comment (MerleMak) [inline components/Loading.js:16]:
```suggestion
export const RotatingSVG = styled(PlateWheel)`
```

File: components/Loading.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import PlateWheel from "/public/icons/svg/plate-wheel.svg";
+
+export default function LoadingComponent() {
+  return <RotatingSVG />;
+}
```
Comment (MerleMak) [inline components/Loading.js]:
```suggestion
```
when the styled component doesn't have any logic attached to it, you can export it directly and don't need to wrap it in a function :)

File: pages/_app.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import GlobalStyle from "../styles";
 import Layout from "@/components/Layout";
+import LoadingComponent from "@/components/Loading";
```
Comment (MerleMak) [inline pages/_app.js]:
```suggestion
```
You don't seem to be using this :)

File: pages/favorites.js [hunk 5]
```diff
@@ -3,6 +3,7 @@ import CardSkeleton from "@/components/Styled/CardSkeleton";
 import MealCard from "@/components/Styled/MealCard";
 import Header from "@/components/Styled/Header";
 import StyledUl from "@/components/StyledUl";
+import LoadingComponent from "@/components/Loading";
```
Comment (MerleMak) [inline pages/favorites.js:6]:
```suggestion
import RotatingSVG from "@/components/Loading";
```

File: pages/favorites.js [hunk 6]
```diff
@@ -28,8 +29,8 @@ export default function Favorites({
         <Header text="Favoriten 🥗" />
         <StyledArticle>
           <StyledUl>
-            <h2>Lade Rezepte...</h2>
-            <CardSkeleton amount={5} $isLoading />
+            <LoadingComponent />
```
Comment (MerleMak) [inline pages/favorites.js]:
```suggestion
            <RotatingSVG />
```

**Review by MerleMak — COMMENTED** [review]
Hi! Nice job, 
Just one note on the styled component. 
I just left you one example. Of course you can also leave the name LoadingComponent and then just rename the styled component itself to LoadingComponent. However you like, just make sure it is coherent :)

Oh an make sure the check don't fail :)

**Comment by avoesen** [comment]
Thank you for the feedback 🥳 I have restructured the component and also added the skeleton loader.

**Pull request merged**